scripts/run-pelorus-e2e-tests

```diff
@@ -169,41 +195,177 @@ sed -i.bak "s/your_org/$demo_org/g" "${DWN_DIR}/mongo-persistent.yaml"
+          secret_filepath=$( mksecret_temp "$PELORUS_NAMESPACE" "$secret_name" "${!env_token_name}" )
+      elif [ -r "${SECRETS_DIR}/${env_token_name}" ] && [ -r "${SECRETS_DIR}/${env_user_name}" ]; then
+          echo "Getting API TOKEN and API USER from secrets mounted directory"
+          secret_filepath=$( mksecret_temp "$PELORUS_NAMESPACE" "$secret_name" "$( cat "${SECRETS_DIR}/${env_token_name}" )" "$( cat "${SECRETS_DIR}/${env_user_name}" )" )
+      elif [ -r "${SECRETS_DIR}/${env_token_name}" ]; then
+          echo "Getting API TOKEN from secrets mounted directory"
+          secret_filepath=$( mksecret_temp "$PELORUS_NAMESPACE" "$secret_name" "$( cat "${SECRETS_DIR}/${env_token_name}" )" )
+      else
+          echo "ERROR: API Token for ${secret_name} not provided, exiting..."
+          exit 1
+      fi
+      # Ensure path is set and file is readable
+      if [[ -n "${secret_filepath}" ]] && [ -r "${secret_filepath}" ]; then
+          oc apply -f "${secret_filepath}"
+          oc_apply=$?
+          if [[ $oc_apply = 0 ]]; then
+              echo "Secret $secret_name was added"
+              SECRET_CONFIGURED=true
+          fi
+          rm "${secret_filepath}"
+      fi
+    fi
+    echo "${SECRET_CONFIGURED}"
+}
+
+# enable the pelorus failure exporter for github
+if [ "${ENABLE_FAIL_EXP}" == true ]; then
+    secret_configured=$(create_k8s_secret github-secret TOKEN)
+    secret_exit=$?
+    echo "${secret_configured}"
+    if [[ $secret_exit != 0 ]]; then
         exit 1
     fi
 
     # uncomment the failure exporter in ci_values.yaml
-    if [ $GITHUB_SECRET_CONFIGURED == true ]; then
+    if [[ "$secret_configured" == *true ]]; then
         sed -i.bak "s/#@//g" "${DWN_DIR}/ci_values.yaml"
         echo "The pelorus failure exporter has been enabled"
     fi
 
     # if required update the failure issue github organization
-    if [ $demo_org != "konveyor" ]; then
+    if [ "$demo_org" != "konveyor" ]; then
         sed -i.bak "s/konveyor\/mig-demo-apps/$demo_org\/mig-demo-apps/g" "${DWN_DIR}/ci_values.yaml"
     fi
 fi
 
+# enable gitlab committime exporter
+if [ "${ENABLE_GITLAB_COM_EXP}" == true ]; then
+    secret_configured=$(create_k8s_secret gitlab-secret GITLAB_API_TOKEN)
+    secret_exit=$?
+    echo "${secret_configured}"
+    if [[ $secret_exit != 0 ]]; then
+        exit 1
+    fi
+    # uncomment the gitlab committime exporter in ci_values.yaml
+    if [[ "$secret_configured" == *true ]]; then
+        sed -i.bak "s/#gitlab-committime@//g" "${DWN_DIR}/ci_values.yaml"
+        echo "The pelorus gitlab committime exporter has been enabled"
+    fi
+
+    # namespace must much one from ci_values.yaml
+    build_ns="gitlab-binary"
+    BINARY_BUILDS_NAMESPACES+=("${build_ns}")
+    build_uri="https://gitlab.com/mpryc/pelorus-gitlab/"
+    build_hash="b807fd8e1b2bd1755eca14960e5352fe89ff466e"
+    eval "${BINARY_BUILD_SCRIPT}" -n "${build_ns}" -b "${build_ns}" -u "${build_uri}" -s "${build_hash}"
```

Comment:
eval isn't necessary here, leading with `"${BINARY_BUILD_SCRIPT}"` should work just fine. Applies to its uses elsewhere also.